### PR TITLE
React 0.14 updates

### PR DESCRIPTION
**Pull request by iamdustan**
Pretty straightforward to handle the new deprecations.

I’m using this branch for the following if you’re interested: :)
https://github.com/iamdustan/component-playground-responsive-iframe/

### CODE REVIEW

**Comment by ryanseddon** [comment]
@iamdustan can you fix up the errors please

**Comment by iamdustan** [comment]
Done and passing.

**Comment by Reggino** [comment]
+1 (Thanks!)

File: index.js [hunk 1]
```diff
@@ -1,6 +1,28 @@
...
+var noop = function() {}
+var swallowInvalidHeadWarning = noop;
+var resetWarnings = noop;
+
+if(hasConsole) {
+  var originalError = console.error;
+  // Rendering a <head> into a body is technically invalid although it
+  // works. We swallow React's validateDOMNesting warning if that is the
+  // message to avoid confusion
+  swallowInvalidHeadWarning = function() {
```
Comment (ryanseddon) [inline index.js]: We'll need to find a better solution to this but for now this is ok I'll create a follow up issue to explore alternatives.

**Comment by ryanseddon** [comment]
Thanks @iamdustan

**Pull request merged**